app/Comment.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
```

Comment:
```suggestion
namespace App/Models;
```
cho vào folder Models